_data/chains/eip155-296.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Hedera Testnet",
+  "chain": "Hedera",
+  "icon": "hedera",
+  "rpc": ["https://testnet.hashio.io/api"],
+  "features": [{ "name": "EIP155" }, { "name": "EIP1559" }],
+  "faucets": ["https://portal.hedera.com"],
+  "nativeCurrency": {
+    "name": "hbar",
+    "symbol": "HBAR",
+    "decimals": 8
+  },
+  "infoURL": "https://hedera.com",
+  "shortName": "hedera-testnet",
+  "chainId": 296,
+  "networkId": 296,
+  "slip44": 3030,
+  "explorers": [
+    {
+      "name": "HashScan",
+      "url": "https://hashscan.io/testnet",
```

Comment:
is 404ing

Reply:
I set the url for the API, I thought it was needed that format reading the EIP3091 specification. You can get results using for example the url: https://hashscan.io/testnet/block/1000

You can get the dashboard homepage with this url: https://hashscan.io/testnet/dashboard 

Should I insert that instead?

Reply:
I think https://hashscan.io/testnet should redirect to https://hashscan.io/testnet/dashboard

Reply:
Ok, understood, I'll suggest them implementing that redirect.

In the meanwhile, to progress the PR and if it is ok for you, I'm going to update the URLs with the "human paths", and set `none` as supported standard. Once they implement the redirect, I will submit a PR updating the URLs and the supported standard.

Does it work for you?

Reply:
yes